src/credentials/CHIPCertFromX509.cpp

```diff
@@ -131,7 +131,7 @@ static CHIP_ERROR ConvertDistinguishedName(ASN1Reader & reader, TLVWriter & writ
                     }
 
                     // If the attribute is a CHIP-defined attribute that contains a 64-bit CHIP id...
-                    if (IsChipIdX509Attr(attrOID))
+                    if (IsChipX509Attr(attrOID))
```

Comment:
This change does not look right.  The intent is that `IsChipX509Attr` tests true for all CHIP attrs and `IsChipIdX509Attr` tests true for the ones that should decode to a 64-bit int...  So the test that used to be here was correct.